to.etc.domui/src/main/java/to/etc/domui/server/ApplicationRequestHandler.java

```diff
@@ -50,7 +50,10 @@
 	private final DomApplication m_application;
 
 	@NonNull
-	private OopsFrameRenderer m_oopsRenderer = new OopsFrameRenderer();
+	private final OopsFrameRenderer m_oopsRenderer = new OopsFrameRenderer();
+
+	@NonNull
+	private final OopsFrameRenderer m_oopsPrdRenderer = new OopsFramePrdRenderer();
```

Comment:
I think this is not a very nice solution. Why not just pass the "testmode" as a parameter to the generation function in the OopsRenderer?